### ISSUE TITLE
Container updates not detected

### ISSUE THREAD

**Issue by jicho** [crazy-max/diun#755]
Hi,

I've installed Diun with this command:
```
docker run -d --name diun \
  -e "TZ=Europe/Amsterdam" \
  -e "LOG_LEVEL=info" \
  -e "LOG_JSON=false" \
  -e "DIUN_WATCH_WORKERS=20" \
  -e "DIUN_WATCH_SCHEDULE=0 5 * * *" \
  -e "DIUN_WATCH_JITTER=30s" \
  -e "DIUN_PROVIDERS_DOCKER=true" \
  -e “DIUN_PROVIDERS_DOCKER_WATCHBYDEFAULT=true” \
  -e "DIUN_NOTIF_MAIL_HOST=127.0.0.1" \
  -e "DIUN_NOTIF_MAIL_PORT=25" \
  -e "DIUN_NOTIF_MAIL_FROM=me@example.com" \
  -e "DIUN_NOTIF_MAIL_TO=me@example.com" \
  -v "/volume1/docker/diun/data:/data" \
  -v "/var/run/docker.sock:/var/run/docker.sock" \
  -l "diun.enable=true" \
  crazymax/diun:latest
```

When I do a mail test everything works with the notifications, the cron-job is up-and-running but in the logs I always get:
`Jobs completed added=0 failed=0 skipped=0 unchanged=13 updated=0`

Looking at this log line:
`INF New image found image=docker.io/n8nio/n8n:latest provider=docker`

Portainer says that I have this image installed:
`n8nio/n8n:latest@sha256:e0ccf8487747bab3c0ef88e2e827a174a69e3959c14f05b7d5453e8aedd9e576`

But the commando `diun image list` it saying (just as the n8n GUI) that there another version:
`sha256:ac8d9499b5cc24f272d4bab8ac60a526c328d31b8e08744ae9caafdcf2c0129e`

Why am I not receiving a notification for the update?
I'm sure that I'm missing something, but I have no clue what :(

Hope someone can help me with this 🤞

**Comment by jicho** [comment]
After doing some more diving into this I've found #751.
Not sure if I have the same issue, so I've added `diun.watch_repo=true` to the n8n image, as a result diun started to send a lot of update emails.

After adding `diun.include_tags=latest` now I'm not getting any emails on latest.

I'm still convinced since I know that are updates for n8n compared with the version that I've running, but what I'm doing wrong.... I have no clue :(
